### PR TITLE
docs: Update README

### LINKED ISSUE #9: Instructions on how to start the project locally is out of date
* Clone url is out of date/invalid
* Might be worthwhile explaining the environment variables needed

https://github.com/gatsbyjs/api.gatsbyjs.org/blob/f2d7f845b162ed5f9d75bc7d5f1c99c31f56bb39/README.md#step-1-start-the-api-locally

### PR DESCRIPTION
**Pull request by PSalant726**
- Update clone URL
- Add link to test `cURL` command
- Format Step 4 heading
- Add Step 3, `env` variables and descriptions

Fixes #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,8 +68,28 @@ To get your JWT token, visit <https://store.gatsbyjs.org> and log in with your G
...
-### Step 3: Send a test request using cURL
+### Step 3: Configure `env` variables
+
+The `.env.EXAMPLE` file contains a list of `env` variables used in various locations throughout the repository. Configure each variable's value per your environment setup and credentials.
+
+| Name                    | Description                                                       |
+| ----------------------- | ----------------------------------------------------------------- |
+| `APP_LOGLEVEL`          | The debug log output level. Default: `info`                       |
+| `APP_HOST`              | The hostname of the API server. Default: `localhost`              |
+| `APP_PORT`              | The port number for the API server. Default: `8080`               |
```
Comment (chadfawcett) [inline README.md]: As far as I can tell, these don't have defaults, they're just pre-populated in the `.env.EXAMPLE`. (ie if you run the app without having a `.env`, these variables will be undefined)

File: README.md [hunk 2]
```diff
@@ -68,8 +68,28 @@ To get your JWT token, visit <https://store.gatsbyjs.org> and log in with your G
...
+| `AUTH0_AUDIENCE`        |                                                                   |
```
Comment (chadfawcett) [inline README.md]:
```suggestion
| `AUTH0_AUDIENCE`        |   The unique identifier for your Auth0 API               |
```



https://auth0.com/docs/quickstart/backend/nodejs#create-an-api

File: README.md [hunk 3]
```diff
@@ -68,8 +68,28 @@ To get your JWT token, visit <https://store.gatsbyjs.org> and log in with your G
...
+| `AUTH0_DOMAIN`          |                                                                   |
```
Comment (chadfawcett) [inline README.md]:
```suggestion
| `AUTH0_DOMAIN`          | Your Auth0 domain               |
```

**Comment by chadfawcett** [comment]
LGTM!

File: README.md [hunk 4]
```diff
@@ -51,8 +51,8 @@ curl -X POST \
 ### Step 1: Start the API locally
 
 ```bash
-git clone git@github.com:ichabod-cranes/mansion.git
-cd mansion/api.gatsbyjs.com
+git clone https://github.com/gatsbyjs/api.gatsbyjs.org.git
```
Comment (andrewnester) [inline README.md]: Minor: I would leave git SSH link intead

**Pull request merged**